cli/install/template.go

```diff
@@ -148,8 +148,6 @@ spec:
         imagePullPolicy: {{.ImagePullPolicy}}
         args:
         - "public-api"
-        - "-addr=:8085"
-        - "-metrics-addr=:9995"
         - "-telemetry-addr=127.0.0.1:8087"
         - "-tap-addr=127.0.0.1:8088"
```

Comment:
Would you mind also removing these client addr flags as well, since these match the default?